e2e/Dockerfile

```diff
@@ -6,15 +6,18 @@ ENV NODE_ENV=test
 WORKDIR /srv/apps/atj-platform
 COPY . .
 RUN corepack enable
-RUN pnpm install --filter=@atj/spotlight --frozen-lockfile
+RUN --mount=type=cache,id=pnpm,target=/pnpm/store pnpm install --frozen-lockfile
 
 FROM base as test
 ENV E2E_ENDPOINT=http://localhost:9090
+ENV CI=true
 RUN npm install -g serve
 EXPOSE 9090
-RUN pnpm build --filter=@atj/spotlight
+EXPOSE 9191
+RUN pnpm build --filter=@atj/spotlight --filter=@atj/design
 WORKDIR ./e2e
-RUN serve ../apps/spotlight/dist -p 9090 & sleep 5; pnpm playwright test;
+RUN serve ../apps/spotlight/dist -p 9090 -L & sleep 5; pnpm playwright test;
+RUN serve ../packages/design/storybook-static -p 9191 -L & sleep 5; pnpm --filter=end-to-end-tests test:storybook --url http://localhost:9191 --config-dir ../packages/design/.storybook/ --browsers firefox chromium
```

Comment:
@danielnaab - changes made. It's ready for you to take a look again. I created a bash script to help run the docker commands during local development that mounts a volume from the host machine so you can run it locally. I opted to keep this out of package.json because it's outside of the node ecosystem due to its dependency on Docker.